csm/core/blogic/const.py

```diff
@@ -326,6 +326,8 @@
 PROCESS_CLUSTER_OPERATION_REQ = 'process_cluster_operation'
 PROCESS_CLUSTER_STATUS_REQ = 'process_cluster_status'
 ShUTDOWN_SIGNAL = 'shutdown_signal'
+MARK_NODE_FAILURE = 'mark_node_failure'
+NOT_DEFINED = 'NOT_DEFINED'
```

Comment:
Remove it

Reply:
Done. Removed NOT_DEFINED = 'NOT_DEFINED'